Cesium.CodeGen/Ir/Declarations/LocalDeclarationInfo.cs

```diff
@@ -231,7 +238,7 @@ private static (IType, string? Identifier) ProcessDirectDeclarator(IDirectDeclar
                     }
                     else
                     {
-                        var constantResult = ConstantEvaluator.TryGetConstantValue(sizeExpr.ToIntermediate(), scope: null);
+                        var constantResult = ConstantEvaluator.TryGetConstantValue(sizeExpr.ToIntermediate(scope), scope);
```

Comment:
That's the change which did the trick

Reply:
Yep, I see. So much work to just provide the scope here. Thanks!